BSD 3-Clause License.md

```diff
@@ -0,0 +1,11 @@
+# WHATWG BSD 3-Clause License
+
+Copyright 2019 WHATWG (Apple, Google, Mozilla, Microsoft).
```

Comment:
```suggestion
Copyright 2020 WHATWG (Apple, Google, Mozilla, Microsoft).
```
Per ongoing discussion in #102 at least. (Still rather we not include it, but creation year is second-best.)